.github/workflows/ci.yml

```
@@ -0,0 +1,35 @@
+name: CI
+
+on:
+  [pull_request, push]
+
+jobs:
+  build:
+    name: Build
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ 'ubuntu-latest', 'macos-latest' ]
+        ocaml-version:
+          - 4.13.1
+          - 4.14.0
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup OCaml ${{ matrix.ocaml-version }}
+        uses: ocaml/setup-ocaml@v2
+        with:
+          ocaml-compiler: ${{ matrix.ocaml-version }}
+          dune-cache: ${{ matrix.os != 'macos-latest' }}
+
+      - name: Install dependencies
+        run: opam install . --deps-only --with-doc --verbose
```

This repo assumes to be built on Nix, so when CI uses opam it needs `opam pin add ocaml-protoc-plugin https://github.com/Nymphium/ocaml-protoc-plugin#create-rpc-path`.
Additionally, it requires `grpcurl`, `protobuf` and `yarn` for tests.
```suggestion
        run: |
          apt install protobuf grpcurl yarn # I don't know ubuntu has these pkgs or its correct name
          opam pin add ocaml-protoc-plugin https://github.com/Nymphium/ocaml-protoc-plugin#create-rpc-path
          opam install . --deps-only --with-doc --verbose
```